pt/index.html

```diff
@@ -0,0 +1,141 @@
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="pt_BR" />
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+    html {
+      height: 100%;
+    }
+    
+    body {
+      min-height: 100%;
+      display: grid;
+      grid-template-rows: 1fr auto;
+
+      margin: 0 auto;
+      max-width: 672px;
+      line-height: 1.6;
+      font-size: 18px;
+      color: #222;
+      background-color: #FAFAFA;
+      padding: 0 10px;
+      font-family: "Open Sans";
+    }
+
+    @media screen and (min-width: 680px) {
+      main {
+        margin: 40px 0;
+      }
+    }
+
+    h1, h2 {
+      line-height: 1.2;
+    }
+
+    footer {
+      grid-row-start: 2;
+      grid-row-end: 3;
+      color: #888;
+      font-size: 9px;
+    }
+
+    a {
+      color: #07A;
+    }
+    a:visited {
+      color: #941352;
+    }
+    footer > a {
+      color: #888 !important;
+    }
+
+    blockquote {
+      background: #F9F9F9;
+      border-left: 10px solid #CCC;
+      margin: 1.5em 10px;
+      padding: 0.5em 10px;
+
+      font-family: monospace;
+    }
+    blockquote p {
+      display: inline;
+    }
+
+    @media screen and (min-width: 400px) {
+      blockquote {
+        display: table;
+      }
+      blockquote > * {
+        display: table-cell;
+      }
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>Não pergunte para pedir, apenas peça</h1>
+    <p>
+      De vez em quando, nas salas de bate-papo online em que eu ando, alguém aparece e diz algo assim
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Algum especialista em Java por aí?
+      </p>
+    </blockquote>
+    <p>
+      Esta é uma má forma, por várias razões. O que a pessoa está realmente
+      <em>perguntando</em> é,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Quaisquer especialistas em Java que estejam dispostos a se comprometer a investigar
+        problema, seja o que for, mesmo que não seja realmente
+        relacionado a Java ou se alguém que não sabe nada sobre Java
+        poderia realmente responder a minha pergunta?
+      </p>
+    </blockquote>
+    <p>
+      Existem muitas razões pelas quais as pessoas que TÊM conhecimento não admitir isso. Ao perguntar, você está pedindo mais do que pensa você está perguntando.
+    </p>
+    <p>
+      Você está pedindo às pessoas que assumam a responsabilidade. Você está questionando as pessoas confiança em suas habilidades. Você também está murando desnecessariamente pessoas fora. Costumo responder perguntas relacionadas a idiomas ou bibliotecas que nunca foram usadas, porque as respostas são (de um modo programador) message
```

Comment:
There's a dangling "message" at the end of the line. Also, this section sounds somewhat too literal.